<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>1</number>
    </property>
    <item>
     <widget class="QTabBar" name="tabCharacters" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="mapAndChatSplitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="TilemapTownMapView" name="tilemapTownMapView" native="true"/>
      <widget class="QSplitter" name="chatViewInputSplitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>3</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <widget class="QTextEdit" name="chatLog">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>3</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="chatBoxAndChannelsLaypit" stretch="0,0">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <widget class="QTabBar" name="tabChatChannels" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="textInput">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="placeholderText">
            <string>chat here...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain">
    <property name="title">
     <string>Tilemap Town</string>
    </property>
    <addaction name="actionConnect_to_a_server"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuServer">
    <property name="title">
     <string>Server</string>
    </property>
    <addaction name="actionDisconnect"/>
    <addaction name="actionReconnect"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionBuild"/>
    <addaction name="actionMail"/>
    <addaction name="actionFileUploads"/>
    <addaction name="actionInventory"/>
    <addaction name="separator"/>
    <addaction name="actionDebug_info"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actioContents"/>
    <addaction name="actionAbout"/>
    <addaction name="actionCheck_for_updates"/>
    <addaction name="separator"/>
    <addaction name="actionSource_code"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuAnimation">
     <property name="title">
      <string>Animation</string>
     </property>
     <addaction name="actionEntity_animation_2"/>
     <addaction name="actionMap_tile_animation"/>
     <addaction name="actionUser_particles"/>
    </widget>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
    <addaction name="actionReset_zoom"/>
    <addaction name="menuAnimation"/>
   </widget>
   <widget class="QMenu" name="menuMap">
    <property name="title">
     <string>Map</string>
    </property>
    <addaction name="actionMapInformation"/>
    <addaction name="actionList_of_users"/>
   </widget>
   <addaction name="menuMain"/>
   <addaction name="menuWindow"/>
   <addaction name="menuMap"/>
   <addaction name="menuSettings"/>
   <addaction name="menuView"/>
   <addaction name="menuServer"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionReconnect">
   <property name="text">
    <string>Reconnect</string>
   </property>
  </action>
  <action name="actionBuild">
   <property name="text">
    <string>Build</string>
   </property>
  </action>
  <action name="actionInventory">
   <property name="text">
    <string>Inventory</string>
   </property>
  </action>
  <action name="actionFileUploads">
   <property name="text">
    <string>File uploads</string>
   </property>
  </action>
  <action name="actionMapInformation">
   <property name="text">
    <string>Map information</string>
   </property>
  </action>
  <action name="actionMail">
   <property name="text">
    <string>Mail</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionConnect_to_a_server">
   <property name="text">
    <string>Connect to a server</string>
   </property>
  </action>
  <action name="actionDebug_info">
   <property name="text">
    <string>Debug info</string>
   </property>
  </action>
  <action name="actioContents">
   <property name="text">
    <string>Contents</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSource_code">
   <property name="text">
    <string>Source code</string>
   </property>
  </action>
  <action name="actionCheck_for_updates">
   <property name="text">
    <string>Check for updates</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionList_of_users">
   <property name="text">
    <string>List of users</string>
   </property>
  </action>
  <action name="actionEntity_animation">
   <property name="text">
    <string>Entity animation</string>
   </property>
  </action>
  <action name="actionMap_animation">
   <property name="text">
    <string>Map animation</string>
   </property>
  </action>
  <action name="actionShow_user_particles">
   <property name="text">
    <string>Show user particles</string>
   </property>
  </action>
  <action name="actionEntity_animation_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Entity animation</string>
   </property>
  </action>
  <action name="actionMap_tile_animation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Map tile animation</string>
   </property>
  </action>
  <action name="actionUser_particles">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>User particles</string>
   </property>
  </action>
  <action name="actionReset_zoom">
   <property name="text">
    <string>Reset zoom</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TilemapTownMapView</class>
   <extends>QWidget</extends>
   <header>tilemaptownmapview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QTabBar</class>
   <extends>QWidget</extends>
   <header location="global">QTabBar</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
